src/axom/slic/docs/sphinx/sections/appendix.rst

```diff
@@ -52,6 +52,66 @@ logging within an application.
   functionality in application specific macros to better suit the requirements
   of the application.
 
+.. _CollectiveSlicMacros:
+
+Collective Slic Macros
+^^^^^^^^^^^^^^^^^^^^^^^^^
+A subset of SLIC macros are collective operations when used with
+MPI-aware :ref:`logStream` instances such as :ref:`SynchronizedStream`
+or :ref:`LumberjackStream`.
+
+Additionally, macros such as ``SLIC_WARNING`` and ``SLIC_CHECK`` become collective
+operations when certain flags are toggled on or functions are called. Other macros
+such as ``SLIC_ERROR`` and ``SLIC_ASSERT`` can be made not collective when certain
+functions are called.
+
+The table below details which SLIC macros are collective:
+
+.. list-table:: Title
+   :widths: 20 20 60
+   :header-rows: 1
+
+   * - Macro
+     - Collective
+     - Notes
+   * - ``SLIC_INFO``
+     - No
+     - 
+   * - ``SLIC_INFO_IF``
+     - No
+     - 
+   * - ``SLIC_ERROR``
+     - Yes
+     - Not collective when ``slic::disableAbortOnError()`` is called
+   * - ``SLIC_ERROR_IF``
+     - Yes
+     - Not collective when ``slic::disableAbortOnError()`` is called
+   * - ``SLIC_WARNING``
+     - Yes (see Note)
```

Comment:
I would remove all the `(see Note)` since there are notes for all of the yes's and its right there.  Though I could be convinced to add them to all of them but my preference is to just remove it.